### PR TITLE
Improvements & SMS

### PR DESCRIPTION
**Pull request by brmzkw**

### CODE REVIEW

**Comment by evadot** [comment]
Could you use print() instead of sys.stdout.write() ?
You can use print("millipede", end="") to remove the newline.

File: millipede/__init__.py [hunk 1]
```diff
@@ -35,6 +38,34 @@ def millipede(size, comment=None, reverse=False):
...
+    It requires to have a contract with Rentabiliweb, valid credentials and to
+    send the message from a whitelisted IP address.
+    """
+    try:
+        import requests
+    except ImportError:
+        print('requests is required to send SMS.', file=sys.stderr)
+        sys.exit(1)
+
+    api_url = 'https://sms.v3.eiole.com/send'
```
Comment (dustbyte) [inline millipede/__init__.py:56]: I think this value should be placed in a configuration management system.

What about etcd?

Comment (evadot) [inline millipede/__init__.py:56]: I would prefer Riak

**Comment by evadot** [comment]
Merged, thanks !!!

**Comment by tusbar** [comment]
the readme is broken

**Pull request merged**